src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4532,6 +4532,8 @@ private StatementAst EnumDefinitionRule(List<AttributeBaseAst> customAttributes,
             //G      enum-member  new-lines:opt
             //G      enum-member-list   enum-member
 
+            Type[] validBaseTypes = new[] { typeof(byte), typeof(sbyte), typeof(short), typeof(ushort), typeof(int), typeof(uint), typeof(long), typeof(ulong) };
```

Comment:
Why not static?
And if we moved this to Parser why we did not remove this from PSType?

Reply:
I think it should be static and we should use consts like TypeCode.Int16 and method type.GetTypeCode().

Reply:
Switching to TypeCode constants sounds like an excellent idea, will try this